backend/passport.js

```diff
@@ -0,0 +1,21 @@
+// const JWTStrategy = require('passport-jwt').Strategy;
```

Comment:
im assuming passport isn't ready, we should of made a pull request without passport commented and then you should have branched off for passport specifically with the code not commented out so we know that branch is working on passport directly. We can leave this for now tho no big deal.